src/index.js

```diff
@@ -1,10 +1,94 @@
-// This is the JavaScript entry file - your code begins here
-// Do not delete or rename this file ********
-
-// An example of how you tell webpack to use a CSS (SCSS) file
+// STYLE IMPORTS
 import './css/base.scss';
 
-// An example of how you tell webpack to use an image (also need to link to it in the index.html)
-import './images/turing-logo.png'
-
+// IMAGE IMPORTS
+// import './images/turing-logo.png'
 console.log('This is the JavaScript entry file - your code begins here.');
+
+// QUERY SELECTORS
+const quoteButton = document.querySelector('.quote-button');
+const closeQuoteButton = document.querySelector('.close');
+const requestButton = document.querySelector('.request-button');
+const dateInput = document.querySelector(".form-date");
+const durationInput = document.querySelector(".form-duration");
+const numTravelersInput = document.querySelector(".form-travelers");
+const destinationInput = document.querySelector(".form-destination");
+
+// JS IMPORTS
+import Traveler from './jsClasses/traveler';
+import { 
+  displayTravelerTrips, 
+  displayTravelerTotal, 
+  displayQuote, 
+  hideQuote,
+} from './domUpdates';
+
+import { 
+  getData,
+  postTrip
+} from './apiRequests';
+
+import DestinationsRepo from './jsClasses/destinationsRepo';
+import TripsRepo from './jsClasses/tripsRepo';
+
+
+// Global Variables
+let tripsRepo;
+let destinationsRepo;
+let traveler;
+
+// Initial Data and DOM Population
+const populateTravelerMain = () => {
+  Promise.all([getTravelerData(),
+    getTripsData(),
+    getDestinationsData()
+  ])
+    .then(response => {
+      traveler = new Traveler(response[0]);
+      tripsRepo = new TripsRepo(response[1]);
+      destinationsRepo = new DestinationsRepo(response[2]);
+      displayTravelerTrips(traveler, tripsRepo, destinationsRepo);
+      displayTravelerTotal(traveler, tripsRepo, destinationsRepo);
+    });
+}
```

Comment:
👏  really clear what you have going on in getting all the data here to display for the traveler